ndcube/mixins/plotting.py

```diff
@@ -50,6 +50,10 @@ def plot(self, axes=None, plot_axes=None, axes_coordinates=None,
 
         wcs: `astropy.wcs.wcsapi.BaseHighLevelWCS`
             The WCS object to define the coordinates of the plot axes.
+
+        kwargs :
+            Any additional keyword arguments are given to
+            `sunpy.visualization.animator.ArrayAnimatorWCS`.
```

Comment:
```suggestion
            Additional keyword arguments are given to the underlying plotting infrastructure
            which depends on the dimensionality of the data and whether 1 or 2 plot_axes are defined.
            Animations: `sunpy.visualization.animator.ArrayAnimatorWCS`
            Static 2-D images: `matplotllib.pyplot.imshow`
            Static 1-D line plots: `matplotllib.pyplot.plot`
```

Reply:
This does not render well: https://sunpy--371.org.readthedocs.build/projects/ndcube/en/371/api/ndcube.mixins.NDCubePlotMixin.html#ndcube.mixins.NDCubePlotMixin

Reply:
Should be fixed into a list now